webapp/src/components/channel_header_button/channel_header_button.tsx

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import './channel_header_button.scss';
+
+type Props = {
+    shouldHighlight: boolean,
+};
+
+export default function ChannelHeaderButton(props: Props) {
+    let btnClass = 'icon fa fa-cloud';
+    if (props.shouldHighlight) {
+        btnClass += ' todo-plugin-icon--active';
+    }
+
+    return (
+        <i className={btnClass}/>
+    );
+}
```

Comment:
There are a few files with a small formatting issue of not having a newline at the end. We generally want the newline, but this can be done in a follow-up.